Bottom: d8d223bf77ea14e3973477682d504ca9b1eda126
Top:    3bebae2763e251d2ba9318e435cb93537e76d30f
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-09 17:07:06 -0500

Add parameter for buildbot version



---
diff --git a/docker/buildbot-worker-ntel/Dockerfile b/docker/buildbot-worker-ntel/Dockerfile
index 05d87e0..11a726f 100644
--- a/docker/buildbot-worker-ntel/Dockerfile
+++ b/docker/buildbot-worker-ntel/Dockerfile
@@ -58,12 +58,13 @@ RUN echo "dash dash/sh boolean false" \
 |   debconf-set-selections -v \
 &&  dpkg-reconfigure -f noninteractive dash
 
+ARG BUILDBOT_VERSION
 # Install required python packages, and twisted
 RUN wget https://bootstrap.pypa.io/get-pip.py \
 &&  python get-pip.py --no-cache-dir \
 &&  pip --no-cache-dir install \
     'twisted[tls]' \
-    'buildbot-worker==0.9.13'
+    buildbot-worker==${BUILDBOT_VERSION}
 
 # Install software for image signing
 RUN curl http://172.16.64.2/~georgem/610-009981-015_SW_PTK_5.3_Client_RevA.tar \
